File: src/main/java/com/p2p/lending/service/CertifrecordService.java
package com.p2p.lending.service;

import java.util.List;
import java.util.Map;

import com.p2p.lending.pojo.Certifrecord;

public interface CertifrecordService {

	public List<Certifrecord> queryCertifrecord(Map<String, Object> map);

	public int updateCertifrecord(Map<String, Object> map);

	public int addCertifrecord(Map<String, Object> map);
}

File: src/main/java/com/p2p/lending/service/CertifrecordService.kt
package com.p2p.lending.service

import com.p2p.lending.entity.Certifrecord

interface CertifrecordService {

    fun queryCertifrecord(map: Map<String, Any>): List<Certifrecord>

    fun updateCertifrecord(map: Map<String, Any>): Int

    fun addCertifrecord(map: Map<String, Any>): Int
}
